omod/src/main/webapp/resources/htmlforms/OBGYN.xml

```diff
@@ -1767,8 +1767,8 @@
 										</tr>
 										<tr class="checkboxGroup">
 											<td>
-												<obsgroup groupingConceptId="CIEL:159947">
-													<span class="titre" ch-target="#cacher31" id="trigger">
+												<obsgroup groupingConceptId="b4c4f7fd-61cd-4c06-89f7-a40478501011">
```

Comment:
@valvijo , I noticed that the answer of the question here is **Pré-éclampsie**  and seems to not be related to the group name  **Visit Diagnoses Retard croissance Intrautérin group** . Can you please verify

Reply:
@valvijo , I  fixed this issue and created more groups to to disaggregate the **Current medication order** group